tests/integration/test_message_embeddings.py

```
@@ -0,0 +1,394 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from src import models
+from src.crud import create_messages, create_messages_for_peer, search
+from src.models import Peer, Workspace
+from src.schemas import MessageCreate
+
+"""
+Tests for message embedding functionality.
+
+These tests verify that message embeddings are created, stored, and can be searched.
+"""
```

Pylint (3.3.7)

[warning] String statement has no effect

(W0105)